BankManagerConsoleApp/BankManagerConsoleApp/Bank.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Bank.swift
+//  BankManagerConsoleApp
+//
+//  Created by mmim, malrang.
+//
+
+import Foundation
+
+struct Bank {
+    private let clerk: Workable
+    private var clientQueue = Queue(list: LinkedList<Client>())
+    private var numberOfClients = 0
+    
+    init(with clerk: Workable) {
+        self.clerk = clerk
+    }
+    
+    mutating func executeBankWork() {
+        let start = CFAbsoluteTimeGetCurrent()
+
+        receiveClients()
+        DispatchQueue.global().sync {
+            serveClients()
+        }
+        
+        let interval = CFAbsoluteTimeGetCurrent() - start
+        let totalTime = String(format: "%.2f", interval)
+        print("업무가 마감되었습니다. 오늘 업무를 처리한 고객은 총 \(numberOfClients)명이며, 총 업무시간은 \(totalTime)입니다.")
```

Comment:
이 문구도 변수로 선언 후에 사용하는건 어떨까요? ㅎㅎ 
이런 형태도 가능할 것 같습니다

```suggestion
        let resultDescription = "업무가 마감되었습니다. 오늘 업무를 처리한 고객은 총 %d명이며, 총 업무시간은 %@입니다."
        let resultMessage = String(format: resultDescription, numberOfClients, totalTime)
        print(resultMessage)
```

Reply:
🥰 정말 저희가 찾던 그 방법입니다!
이 부분은 좀 더 공부해보겠습니다!